<launch>

  <node pkg="ccny_rgbd" type="sparse_tracker_node" name="sparse_tracker" 
    output="screen">

    <param name="base_frame"        value="camera_link"/>
    <param name="fixed_frame"       value="odom"/>
    <param name="history_size"      value="5"/>
       
    <param name="feature/detector_type"       value="3"/>
    <param name="feature/smooth"              value="0"/>
    <param name="feature/max_range"           value="7.0"/>
    <param name="feature/compute_descriptors" value="false"/>

    <param name="reg/type"                    value="0"/>
    <param name="reg/max_iterations"          value="25"/>
    <param name="reg/max_corresp_dist"        value="0.15"/>
    <param name="reg/tf_epsilon"              value="1e-15"/>
    <param name="reg/use_ransac_rejection"    value="true"/>
    <param name="reg/ransac_inlier_threshold" value="0.15"/>

    <param name="gicp/max_iteration_inner"   value="20"/>

  </node>

  <node pkg="ccny_3dmap_server" type="map_server_node" name="map_server" 
    output="screen">

    <param name="fixed_frame"  value="odom"/>
    <param name="resolution"   value="0.03"/>

    <param name="max_range"    value="5"/>

  </node>

</launch>

